src/Modix.Services/Moderation/MessageContentCheckBehaviour.cs

```diff
@@ -43,6 +44,32 @@ public class MessageContentCheckBehaviour :
         public Task HandleNotificationAsync(MessageUpdatedNotification notification,
             CancellationToken cancellationToken = default)
             => TryCheckMessageAsync(notification.NewMessage);
+        public async Task HandleNotificationAsync(VoiceChannelStatusUpdatedNotification notification,
+            CancellationToken cancellationToken = default)
+        {
+            var channel = await notification.Channel.GetOrDownloadAsync();
+            var newStatus = notification.NewStatus;
+            var isBlocked = await IsContentBlocked(channel, newStatus);
+
+            if (!isBlocked)
+            {
+                return;
+            }
+
+            //TODO: note infraction for change author
+            //How do we obtain the user who changed the status?
```

Comment:
We do not need a todo here, as this wouuldn't be an infraction.

Reply:
removed the comment